src/features/rewards/modalBackupRestore/index.tsx

```diff
@@ -33,10 +37,12 @@ export interface Props {
   error?: React.ReactNode
   id?: string
   testId?: string
+  funds?: string
```

Comment:
We need to be able to produce a special warning for restore if the user has funds in their wallet.

What should be fed in this prop from assembly is a formatted string of the users BAT balance. Ex: '55 BAT'